bookShop/src/app/modules/components/book/book.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { BookComponentComponent } from './book-component/book-component.component';
+
+@NgModule({
+  declarations: [
+      BookComponentComponent
+  ],
+  imports: [
+    BrowserModule,
+    FormsModule,
+  ],
+  exports: [
+    BookComponentComponent
+  ],
+  providers: [],
+  bootstrap: []
```

Comment:
You don't need a bootstrap module. It uses only in AppModule

Reply:
Corrected